src/zope/app/container/browser/tests/test_adding.py

```diff
@@ -13,46 +13,49 @@
 ##############################################################################
 """Adding implementation tests
 """
+from zope.testing import renormalizing
+import re
 import doctest
 import unittest
 
 import zope.interface
 import zope.security.checker
+from zope.browsermenu.interfaces import AddMenu
+from zope.browsermenu.interfaces import IMenuItemType, IBrowserMenu
```

Comment:
We do not have `isort` activated here for now, or am I missing some configuration?

Reply:
No we haven't. I think this was done by `autopep8`.